roles/nginxplus/files/conf/http/dev/templates/cdh-errors.conf

```diff
@@ -0,0 +1,25 @@
+error_page 400 401 402 405 406 407 408 409 410 411 412 413 414 415 416 417 418 421 422 423 424 426 428 429 431 451 501 502 503 504 505 506 507 508 510 511 /cdh-error.html;
```

Comment:
Same thing here. It seems like we need this change, and it needed to go in to test the Solr stuff, but Future Me is going to be happier if we have this change in a PR I can find by searching for "error message" or "CDH error" or something.